rubocop-rails.gemspec

```diff
@@ -33,6 +34,8 @@ Gem::Specification.new do |s|
 
   # Rack::Utils::SYMBOL_TO_STATUS_CODE, which is used by HttpStatus cop, was
   # introduced in rack 1.1
+  s.add_runtime_dependency 'activesupport'
```

Comment:
This dependency should be added above the comment for Rack :-)

Reply:
I did not confirm the result of sorting the dependencies by `rubocop -a`. :see_no_evil: